pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -301,40 +316,53 @@ func errorString(errors interface{}) string {
 	return strings.Join(output, ",")
 }
 
-func (policy *regoEnforcer) getReasonNotAllowed(enforcementPoint string, input inputData) error {
-	inputJSON, err := json.Marshal(policy.redactSensitiveData(input))
+func (policy *regoEnforcer) denyWithError(ctx context.Context, policyError error, input inputData) error {
+	policyDecision := map[string]interface{}{
+		"input":       input,
```

Comment:
should we redact sensitive data here?

Reply:
Yes, we absolutely need to redact sensitive data.